Port constraint_sampler_manager_loader to ROS2

anasarrak opened the pull request.

anasarrak committed:
Port constraint_sampler_manager_loader

anasarrak committed:
Apply clang-format

vmayoral commented:
CI failed @anasarrak, can you review please? @ahcorde can you help?

LanderU commented on moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp:
```diff
@@ -70,27 +74,28 @@ class ConstraintSamplerManagerLoader::Helper
-
-ConstraintSamplerManagerLoader::ConstraintSamplerManagerLoader(
-    const constraint_samplers::ConstraintSamplerManagerPtr& csm)
+node_->get_logger(),
```
This caused the error on the CI:

```bash
/root/moveit2/moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp:93:1: error: ‘node_’ does not name a type; did you mean ‘mode_t’?
 node_->get_logger(),
```

ahcorde committed:
fix problem with CI

vmayoral reviewed (approved):
Let’s wait for the CI before merging but LTGM from my side.

ahcorde commented:
Linux is passing but OSX is failing

anasarrak commented:
OSX is failing because of [this](https://github.com/AcutronicRobotics/geometric_shapes/blob/e02b97df2fa02dedf2f45cd10f3fe47cfd546b63/CMakeLists.txt#L79) line on geoetric_shapes

ahcorde committed:
resource retriever from sources

Update osx-compile-moveit2.sh

Update osx-install-ros2-binary.sh

Update moveit2.repos

Update package.xml

Update osx-compile-moveit2.sh

ahcorde committed:
resource retriever from soruces

ahcorde committed:
Merge pull request #66 from AcutronicRobotics/constraint_sampler_manager_loader_osx

resource retriever from soruces

LanderU committed:
Restore moveit2.repos and remove it from OSX

ahcorde committed:
Update osx-compile-moveit2.sh
